lib/plugins/aws/invokeLocal/index.js

```diff
@@ -107,8 +109,26 @@ class AwsInvokeLocal {
         this.options.data);
     }
 
+    if (runtime === 'python2.7') {
+      return this.invokeLocalPython(
+        handlerPath,
+        handlerName,
+        this.options.data);
+    }
+
     throw new this.serverless.classes
-      .Error('You can only invoke Node.js functions locally.');
+      .Error('You can only invoke Node.js & Python functions locally.');
+  }
+
+  invokeLocalPython(handlerPath, handlerName, event) {
+    return new BbPromise(resolve => {
+      const python = spawn(path.join(__dirname, 'invoke.py'), [handlerPath, handlerName]);
```

Comment:
I haven't tried this, but by spawning, do we still have access to the env vars in the process? This is essential because we load env vars in the `loadEnvVars()` method prior to running the python function.

In other words, **does this python invoke local support supports AWS Lambda env vars feature?**